java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1913,27 +1914,52 @@ public ColumnVector asTimestamp(DType timestampType, String format) {
 
   /**
    * Cast to Strings.
+   * Negative timestamps are not supported
```

Comment:
Could you elaborate here?  A timestamp from before the epoch will be negative.  Will those timestamps somehow not work properly?  If not, why not?

Ah, I see the PR discussion now about negative timestamps.  Please link to the relevant cudf issue in the comment so it's clearer what's going on with negative timestamps.